site/src/patterns/timeline.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Timeline
+description: Common timeline patterns in the Atlas Design System
+template: standard
+---
+
+# Timeline
+
+The timeline pattern is used to display a list of events.
+
+```html
+<div class="border-md border-radius-lg padding-md font-size-md line-height-md">
+	<div class="display-flex justify-content-space-between margin-bottom-sm">
+		<h4 class="font-size-h5">Timeline History</h4>
+		<button class="button button-clear">X</button>
+	</div>
+	<ul class="margin-left-sm">
```

Comment:
Because the order does matter in a timeline, I think this semantically should be an `ol`

Reply:
updated